src/System.Data.Common/src/System/Data/Common/DbColumn.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace System.Data.Common
+{
+    public class DbColumn
+    {
+        private Dictionary<string, object> s_customValues = new Dictionary<string, object>();
```

Comment:
Nit: This should be named `_customValues` unless it's intended to be `static`. Also, `readonly`?